assets/src/application/ui/layout/entityList/pagination/useEntityPagination.js

```diff
@@ -14,24 +14,22 @@ const EMPTY_ARRAY = [];
  *                  - setCurrentPage - callback for changing currentPage
```

Comment:
I kept this file around because there might be some logic extraction needed.

Reply:
we will need this for tracking state for paginated lists no?

Reply:
yup, there is `onChange` prop that we might need to use at some point, this is why I left this file;
for sure it will need to be refactored, so I didn't touch anything yet